core/src/main/scala/com/softwaremill/events/EventMachine.scala

```diff
@@ -45,6 +46,24 @@ class EventMachine(
       })
   }
 
+  def failOverFromStoredEventsTo(timeLimit: OffsetDateTime = OffsetDateTime.now()): Unit = {
```

Comment:
nevermind, you are right it's ok as-is. A minor thing, if you include `To` in the method name, the time limit probably should be mandatory (w/o a default value), otherwise it reads in a weird way ;)